Client/Frontend/Browser/Browser.swift

```diff
@@ -153,10 +159,36 @@ class Browser: NSObject, WKScriptMessageHandler {
             webView.alpha = 1.0
         }
     }
+
+    var bars = [SnackBar]()
+    func addSnackbar(bar: SnackBar) {
+        bars.append(bar)
+        delegate?.snackbarAdded(bar)
+    }
+
+    func removeSnackbar(bar: SnackBar) {
+        if let index = find(bars, bar) {
+            bars.removeAtIndex(index)
+            delegate?.snackbarRemoved(bar)
+        }
+    }
+
+    func removeAllSnackbars() {
+        for bar in bars {
```

Comment:
I worried about this but didn't run into problems. Maybe I was just lucky though. Digging for some info...